frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,46 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+<div class="flex items-start">
+  <div class="mx-10">
+    <img
+      src="https://upload.wikimedia.org/wikipedia/commons/2/2c/Default_pfp.svg"
+      alt="Dummy Profile Picture"
+      class="h-28 w-28"
+    />
+  </div>
+  <h2>Profile of {{ user?.name }}</h2>
+</div>
+<div class="flex items-start">
+  <div class="ml-4 mt-5 grow">
+    <div *ngIf="userId !== userService.user?.id; else ownPageBlock">
+      <div>
+        <h3>Common Projects</h3>
+        <div class="grid grid-cols-4 gap-4">
+          <ng-container
+            *ngIf="(commonProjects$ | async)?.length; else elseBlock"
+          >
+            <mat-card
+              *ngFor="let project of commonProjects$ | async"
+              class="!ml-0 cursor-pointer"
+              [routerLink]="['/project', project.slug]"
+            >
+              <mat-card-content>
+                <b>{{ project.name }}</b
+                ><br />
+                {{ project.description }}
+              </mat-card-content>
+            </mat-card>
+          </ng-container>
```

Comment:
I mostly just wanted to try out Angular 17's new control flow (😄) but I had a few suggestions to fill it out, and keep the text from overflowing in any direction on mobile if you're interested: 
<img width="328" alt="mobile_shared_projects" src="https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/067b2dbb-e1a9-462a-9fc8-09cf75d8a836">

```suggestion
<mat-divider></mat-divider>
        <div class="grid grid-cols-2 gap-4 md:grid-cols-4">
          <ng-container
            *ngIf="(commonProjects$ | async)?.length; else elseBlock"
          >
            <mat-card
              *ngFor="let project of commonProjects$ | async"
              class="!ml-0 cursor-pointer"
              [routerLink]="['/project', project.slug]"
            >
              <mat-card-content>
                <b class="line-clamp-1">{{ project.name }}</b
                ><br />
                @if (project.description) {
                <span class="line-clamp-3 md:line-clamp-5">{{
                  project.description
                }}</span>
                } @else {
                <i class="line-clamp-3 md:line-clamp-5"
                  >No description provided</i
                >
                }
              </mat-card-content>
            </mat-card>
          </ng-container>
```

Reply:
thanks for the comment! I incorporated the feedback!